packages/pipelines/package.json

```diff
@@ -52,12 +54,7 @@
     "typescript": "^5.3.3"
   },
   "peerDependencies": {
-    "react": "^18.2.0",
-    "@testing-library/jest-dom": "^6.2.0",
-    "@patternfly/react-core": "^5.1.2",
-    "@patternfly/react-icons": "^5.1.2",
-    "@patternfly/react-table": "^5.1.2",
-    "@patternfly/react-tokens": "^5.1.2"
```

Comment:
These should still be in peerDependencies as the main application might bring in these as well.